sklearn/utils/testing.py

```diff
@@ -688,6 +692,29 @@ def func(*args, **kwargs):
     return decorator
 
 
+def if_safe_multiprocessing_with_blas(func):
```

Comment:
It's still tested under Python 3.4 and on appveyor. The problem is that there is no way to test when there could be a problem otherwise.

If you want we can restore the test under Linux assuming that Linux users will always use there system BLAS and as far as I know all BLAS versions (OpenBLAS and ATLAS) shipped by Linux distros are fork-safe although I am not 100% sure. 

But if someone compiles there own OpenBLAS with openmp enabled under Linux, that will crash and there is no robust way to test that.